packages/wdio-sync/src/index.ts

```diff
@@ -63,8 +81,8 @@ async function executeSync(fn, retries = defaultRetries, args = []) {
 /**
  * run hook or spec via executeSync
  */
-function runSync(fn, repeatTest = 0, args = []) {
-    return (resolve, reject) =>
+function runSync(this: unknown, fn: Function, repeatTest = defaultRetries, args: unknown[] = []) {
+    return (resolve: any, reject: any) =>
```

Comment:
I guess `any` can be replaced with `Function`?

Reply:
Unfortunately no, and I was not able to find out which one it is for sure from the context... If you have a suggestion you're more than welcome!